docs/running-on-mesos.md

```diff
@@ -194,6 +194,10 @@ conf.set("spark.mesos.constraints", "tachyon=true;us-east-1=false")
 
 For example, Let's say `spark.mesos.constraints` is set to `tachyon=true;us-east-1=false`, then the resource offers will be checked to see if they meet both these constraints and only then will be accepted to start new executors.
 
+Spark rejects offers for slaves with unmet constraints for 120 seconds by default.
+This makes mesos not sending the rejected offers again and again.
+The reject offer duration is configurable using `conf.set("spark.mesos.rejectOfferDurationForUnmetConstraints", "60")` (for example).
+
```

Comment:
I actually don't think this is worth documenting. I can't imagine anyone who wants to set this explicitly as long as we pick a sane enough default.